login.css

```diff
@@ -171,8 +188,15 @@ p{
   	top: 240px;
 }
 
+
 #update{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
+	font-family: 'Roboto', sans-serif;
```

Comment:
13/08/2021: I set the fonts for the text for the forgot password link to roboto and because it is normal text I set the font weight to 300.